vendor/views/mixin.py

```diff
@@ -37,23 +37,23 @@ def user_has_product(self):
 
         TODO: move this to some kind of caching or session variable to avoid hitting the DB on every request.
         """
-
+        site = Site.objects.get_current()
```

Comment:
Site.objects.get_current() is dependent on the SITE_ID to work.

Reply:
Added SetSiteToRequestMixin